pkgs/top-level/aliases.nix

```diff
@@ -1933,7 +1933,6 @@ mapAliases ({
   zinc = zincsearch; # Added 2023-05-28
   zq = zed.overrideAttrs (old: { meta = old.meta // { mainProgram = "zq"; }; }); # Added 2023-02-06
 
-  # TODO(ekleog): add ‘wasm’ alias to ‘ocamlPackages.wasm’ after 19.03
   # branch-off
 
```

Comment:
```suggestion
```

Reply:
The section above  is arranged in dictionary order, but the following part is not

Reply:
True, but these two lines clearly form a phrase. Perhaps leave the `#` there?

Reply:
Maybe `### UNSORTED ###` ？